client/client.cpp

```diff
@@ -0,0 +1,168 @@
+#include "client.h"
+#include <QtWidgets>
+#include <QtNetwork>
+#include <QDebug>
+
+Client::Client(QWidget *parent)
+    : QDialog(parent)
+    ,hostCombo (new QComboBox)
+    ,portLineEdit(new QLineEdit)
+    ,getButton(new QPushButton(tr("Get ")))
+    , tcpSocket(new QTcpSocket(this))
+{
+    setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+    hostCombo->setEditable(true);
```

Comment:
what is this combo for? I remember we specified the server ip manually

Reply:
Overall listing local hosts and ips in the client is useless, we don't want to use local ip as server (in future)